drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CheatSheetView.java

```diff
@@ -24,6 +24,8 @@
 
     void setDMNCheatSheetContent();
 
+    void reset();
```

Comment:
@yesamer 
If possible, declare this method inside _SubDockView_ so that it get's inherited automatically by all specific interfaces

Reply:
Done